builds/jazz-build-module/sonar-module.groovy

```diff
@@ -137,6 +164,113 @@ def cleanUpWorkspace() {
 	}
 }
 
+/**
+* Run Unit Tests and generate Code Coverage Reports
+ */
+def generateCoverageReports() {
+	def domain = service_config['domain']
+    def repo_name = domain + "_" + service_config['service']
```

Comment:
Thee two lines can be included in one line y using service_config['domain']

Reply:
Done